roles/dhcp/handlers/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# handlers file for dhcp-config
+- name: 'dhcp-config cleanup temp'
+  delegate_to: localhost
+  file:
+    path: '{{ dhcp_config_dir.path }}'
+    state: absent
+
+- name: 'cleanup dhcp tmp file'
+  file:
+    path: '/tmp/{{ dhcp_config_temp_file }}'
+    state: absent
+
+- name: 'enable and start dhcp services'
+  service:
+    name: '{{ item }}'
+    enabled: yes
+    state: started
+  with_items:
+  - dhcpd
+  - firewalld
```

Comment:
it probably doesn't hurt to include `firewalld` here, but it's already done as part of the prereq steps and we don't need to reload/restart firewalld